app/src/main/res/layout/view_card_layout.xml

```diff
@@ -12,6 +12,7 @@
         android:text="this is a card!"
         android:textAlignment="center"
         android:gravity="center"
-        style="@style/CardStyle" />
+        style="@style/CardStyle"
```

Comment:
Based on what we read in the Android best practices I think we should look further into the styles we have now and clean them up as well. I can see at a first glance we are using stuff like colorPrimary when we should really be labeling things with their actual color. I'm ok with something like primaryBlue so we know it should be our goto blue color but also can tell what color it is without looking at the definition of the color.

Reply:
https://medium.com/google-developers/theming-with-appcompat-1a292b754b35